app/views/pages/medicaid/amounts_loan_interest.erb

```diff
@@ -0,0 +1,25 @@
+<div class="step-header">
+  <a href="/pages/medicaid/amounts_other_income" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Income & Expense Amounts
+  </h4>
+</div>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Tell us more about the specifics of your student loan interest.</div>
```

Comment:
a thought I've been having as I review these pages: feels a bit confusing to me that so many of these questions do not have a question mark.

Reply:
also, the "tell us more" part feels like it would be asking for qualitative rather than quantitative answers

Reply:
@jessieay agreed. Adding these comments to the Trello to discuss and update copy there